config/locales/es.yml

```diff
@@ -593,23 +594,49 @@ es:
   models:
     intake:
       your_spouse: Su cónyuge
+  portal:
+    client_logins:
+      form:
+        errors:
+          at_least_one: Complete una de las dos entradas.
+          bad_confirmation_number: Número de confirmación incorrecto. Después de 5 intentos fallidos, las cuentas se bloquean.
+          bad_last_four: "TODO Incorrect last four. After 5 failed attempts, accounts are locked."
+      new:
+        title: Le mandaremos un enlace seguro para que pueda ver su progreso.
+      link_sent:
+        link_sent: "Para proteger su identidad, le enviamos un enlace seguro a su información de contacto."
+        title: "Para continuar, por favor visite su enlace seguro."
+      show:
+        title: "TODO Authentication needed to continue."
```

Comment:
FYI I personally left ES translations as TODO yesterday and this got missed when it got picked up after I left. It'll smart one day to get these translations in. cc: @bengolder

Reply:
oof, sorry about that!